src/utils/multiple-instances-warning.ts

```diff
@@ -0,0 +1,20 @@
+interface Global {
```

Comment:
Use the following to augment the Global interface (using symbols, see other comment):

```typescript
import { sherlockInstances } from '../symbols';

declare global {
    namespace NodeJS {
        interface Global {
            [sherlockInstances]: number;
        }
    }
}
```

But when using symbols you do not need to augment the interface. That's ok with me.